template.tpl

```diff
@@ -633,6 +724,18 @@ const getTimestamp = (tagConfig) => {
   }
 };
 
+/**
+ * This function returns the byte size of a UTF-8 string.
+ * https://gist.github.com/mathiasbynens/1010324
+ *
+ * @param {string} s - The string to count the bytes of
+ * @returns {number} Size of s in bytes
+ */
+function byteCount(s) {
+  const rexp = createRegex('%[A-F0-9]{2}', 'g');
+  return encodeUriComponent(s).replace(rexp, 'x').length;
+}
```

Comment:
Do we know if this is required? Google feel like the kind of people who would always use ascii-safe JSON. (`\u` escapes, etc).

Reply:
> Google feel like the kind of people who would always use ascii-safe JSON. 

Can you please explain a bit more? Is there something we can safely assume?

Reply:
Even if the JS value contains unicode characters, when encoding as JSON it's possible to not include the raw unicode bytes, you can alternatively represent those same bytes as the codepoint representation `\uXXXX` (X is hex, [0-9a-f]), so theoretically every unicode-JSON document can be represented as pure ASCII with one byte per character in the escaped format. This allows e.g. the [`ensure_ascii` mode in Python's json module](https://docs.python.org/3/library/json.html#json.dump). So the question is if you pass unicode into GTM-SS's `JSON.stringify()` does it give you the UTF8 bytes with unicode literals, or does it escape non-ascii bytes so we can assume `.length` is correct without needing to check the UTF8 size.

Reply:
Unfortunately:

```
const t = 'abc日本語ée';

logToConsole(json.stringify(t).length);   // 10 
logToConsole(byteCount(t));                     // 15
```

Reply:
Welp, now we know. 😅 Thanks for testing!